src/views/Correction/BcCorrection.vue

```diff
@@ -77,12 +77,12 @@ import { BcCorrectionResource, BenCorrectionResource, CccCorrectionResource, Ulc
     CompletingParty
   }
 })
-export default class BaseCorrection extends Mixins(CommonMixin, DateMixin, FeeMixin, FilingTemplateMixin) {
+export default class BcCorrection extends Mixins(CommonMixin, DateMixin, FeeMixin, FilingTemplateMixin) {
```

Comment:
Hopefully this won't confuse anyone. It's the correction component for "BC corp class" businesses.

Reply:
If there's a risk of confusing future custodians of the code, maybe put a comment in to explain what you've just stated above?

Reply:
Good call. Upated.